public/assets/scss/_index.scss

```diff
@@ -0,0 +1,488 @@
+        width:100%;
+        text-align:center;
+    }
+}
+.arrow {
+    width: 1.5rem;
+    height: 1.5rem;
+}
+.prev {
+    display: block;
+    position: absolute;
+    z-index: 1000;
+    top:50%;
+    transform: translateY(-50%);
+}  
+.next {
+    display: block;
+    position: absolute;
+    right: 0;
+    top:50%;
+    transform: translateY(-50%);
+    z-index: 1000;
+}
+.slide {
+    background-color: $white;
+    border-radius: 0.3rem;
+    margin: 1rem 3rem;
+    padding: 4rem 1rem 2rem 1rem;
+    height: 23rem;
+
+    &-image {
+        width: 5.5rem;
+        height: 5rem;
+        margin-left: auto;
+        margin-right: auto;
+        display: block;
+    }
+    &-header {
+        color: $purple;
+        font-size: 1.2rem;
+        font-weight: 400;
+        padding-top: 1rem;
+        padding-bottom: 1rem;
+    }
+    &-body {
+        color: $gray-700;
+        font-weight: 300;
+        font-size: 0.8rem;
+    }
+}
+.works {
+    padding: 2rem;
+}
+.phone {
+    height: 38rem;
+    width: 30rem;
+
+    &-div {
+        padding-left: 6rem;
+        padding-top: 4rem;
+    }
+}
+.how {
+    &-heading {
+        @extend %heading-font-stack;
+        padding-top: 2rem;
+        padding-bottom: 2rem;
+    }
+    &-head {
+        color: $purple;
+        font-size: 1.4rem;
+        font-weight: 400;
+    }
+    &-text {
+        font-weight: 300;
+        font-size: 1rem;
+        color: $gray-700;
+    }
+}
+.stage-div {
+    padding-right: 1.8rem;
+    padding-bottom: 2rem;
+}
+.bold-letter {
+    font-size: 2.5rem;
+    font-weight: 700;
+}
+.circle {
+    &-line {
+        padding-top: 1.3rem;
+    }
+    &-line-img {
+        height: 8rem;
+    }
+    &-img {
+        height: 1.25rem;
+    }
+}
+.faq {
+    padding-top: 3rem;
+    padding-left: 6rem;
+    padding-bottom: 4rem;
+
+    &-heading {
+        @extend %heading-font-stack;
+        font-weight: 500;
+        color: $blue;
+    }
+    &-card-heading {
+        color: $blue;
+        font-size: 1.2rem;
+        font-weight: 400;
+    }
+}
+.questions {
+    float: right;
+    padding-right: 7rem;
+    padding-top: 1rem;
+    padding-bottom: 2rem;  
+}
+.icon-toggle {
+    float: right;
+}
+.faq-icon {
+    font-size: 4rem;
+    color: $purple;
+}
+.card {
+    border: 0;
+    border-radius: 0;
+    border-bottom: 1px solid $gray-750;
+    padding: 1.5rem 0;
+    margin-left: -3rem;
+
+    &-header {
+        background-color: $white;
+        border: 0;
+    }
+}
+h5 .btn-link {
+    font-size: 1.3rem;
+    font-weight: 400;
+    color: $blue;
+}
+.gallery {
+    background-image: url('../img/backgrounds/deep-dive.png');
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+    text-align: center;
+    padding: 5rem;
+
+    &-heading {
+        @extend %heading-font-stack;
+        padding-top: 2.5rem;
+        padding-bottom: 3rem;
+    }
+}
+.break {
+    display: inline-block;
+}
+
+/* big tablets */
+@media only screen and (max-width: 1024px) {
```

Comment:
Since we are using SASS, it will be best to make use of its best features. Can you move all the styles inline as explained in this comment? https://github.com/Highrachy/ballers-ws/pull/12#discussion_r408684581

Reply:
Sorry I can't. The code is easier for me to read this way.

Reply:
@tobieeniafe Thank you for the kind response. However, when working on a team, decision making is not about 1 person, good decisions are made for the whole team would gain from. It is about making the life of future developers that would work on this easier. Saying that you are not updating a code because it is easier for you to read is an egocentric way of thinking and not favorable for team works.  

The suggested way is more productive because it ensures that the styles can be compared in one selector without jumping from one line to another. It is also easier to update the responsive styles if they are in one selector. This is a win-win situation for everyone and I believe it is a better reason than making it easier for you to read.  Feel free to adopt whatever style favors you for your personal, however, when it comes to teamwork, let's strive to use the best standard always.

Apart from it being easier for you to read (which I don't agree), can you state the technical advantages (in respect to the team) on how your method is more favorable than the suggested one?